Formula/nuclei.rb

```diff
@@ -0,0 +1,46 @@
+class Nuclei < Formula
+  desc "HTTP/DNS scanner configurable via YAML templates"
+  homepage "https://nuclei.projectdiscovery.io/"
+  url "https://github.com/projectdiscovery/nuclei/archive/v2.2.0.tar.gz"
+  sha256 "c503b565230f5375339c72872d496137abf8be26c156e20447d1ae0752169e40"
+  license "MIT"
+  head "https://github.com/projectdiscovery/nuclei.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GO111MODULE"] = "on"
```

Comment:
This shouldn't be needed in modern go builds

Reply:
removed